run.sh

```diff
@@ -3,27 +3,35 @@ if [ ! -n "$WERCKER_NEWRELIC_DEPLOYMENT_API_KEY" ]; then
   exit 1
 fi
 
-if [ ! -n "$WERCKER_NEWRELIC_DEPLOYMENT_APP_NAME" ]; then
-  error 'Please specify app_name property'
+if [ ! -n "$WERCKER_NEWRELIC_DEPLOYMENT_APP_ID" ]; then
+  error 'Please specify app_id property'
   exit 1
 fi
 
 if [ ! -n "$WERCKER_NEWRELIC_DEPLOYMENT_REVISION" ]; then
-  export WERCKER_NEWRELIC_DEPLOYMENT_REVISION="$WERCKER_GIT_BRANCH/${WERCKER_GIT_COMMIT:0:10} by $WERCKER_STARTED_BY"
+  export WERCKER_NEWRELIC_DEPLOYMENT_REVISION="${WERCKER_GIT_BRANCH}/${WERCKER_GIT_COMMIT}"
 fi
 
-NEWRELIC_API="https://api.newrelic.com"
-NR_API_DEPLOYMENTS="$NEWRELIC_API/deployments.xml"
-
-NR_API_VAR="x-api-key"
-NR_DEPLOY_APP="deployment[app_name]"
-NR_DEPLOY_REV="deployment[revision]"
-NR_DEPLOY_SKIP="deployment[skip]"
+if [ ! -n "$WERCKER_NEWRELIC_DEPLOYMENT_ON" ]; then
```

Comment:
Copied this from the https://github.com/investtools/wercker-step-rollbar-notify/blob/master/run.sh